components/resources/wallet_strings.grdp

```diff
@@ -1041,4 +1041,32 @@
   <message name="IDS_BRAVE_WALLET_UNTRUSTED_REQUEST_WARNING" desc="Warning that the pending request may be malicious">This request appears malicious, and may leave you vulnerable to theft. Do not continue unless you trust the app.</message>
   <message name="IDS_BRAVE_WALLET_PROCEED_BUTTON" desc="Proceed action button text">Proceed</message>
   <message name="IDS_BRAVE_WALLET_FOUND_ISSUES" desc="Notice that a number of issues were found">We found <ph name="VALUE">$1<ex>2</ex></ph> issues.</message>
+  <message name="IDS_BRAVE_WALLET_BUTTON_RETRY" desc="Retry an action">Retry</message>
+  <message name="IDS_BRAVE_WALLET_TRANSACTION_PREVIEW_FAILED" desc="Explaination that simulation of a transaction has failed">Transaction preview failed.</message>
+  <message name="IDS_BRAVE_WALLET_ESTIMATED_BALANCE_CHANGE" desc="Grouping header for estimated balance changes">Estimated balance change</message>
+  <message name="IDS_BRAVE_WALLET_APPROVAL_DETAILS" desc="Grouping header for estimated token approval details">Approval details</message>
+  <message name="IDS_BRAVE_WALLET_AUTHORITY_CHANGE" desc="Grouping header for estimated authority changes">Authority change</message>
+  <message name="IDS_BRAVE_WALLET_SCANNING_TRANSACTION" desc="Indication that a transaction is being scanned or simulated">Scanning transaction</message>
+  <message name="IDS_BRAVE_WALLET_STAKER" desc="The address that is staking tokens">Staker</message>
+  <message name="IDS_BRAVE_WALLET_WITHDRAWER" desc="The address that is withdrawing funds">Withdrawer</message>
+  <message name="IDS_BRAVE_WALLET_SIMULATION_WARNING_APPROVAL_TO_EOA" desc="Warning that token transfer permission is being given to someone else">You’re giving someone else permission to transfer tokens on your behalf.</message>
+  <message name="IDS_BRAVE_WALLET_SIMULATION_WARNING_BULK_APPROVALS_REQUEST" desc="Warning that many tokens will be allowed to be transfered">This requests permission to transfer many tokens, which is usually an attempt at theft.</message>
+  <message name="IDS_BRAVE_WALLET_SIMULATION_WARNING_COPY_CAT_DOMAIN" desc="Warning that the dapp may be a malicious copy">This site may be a fake</message>
```

Comment:
nit: needs a dot at the end to be consistent with other warnings

Reply:
Thanks, fixed. [bce6b6f](https://github.com/brave/brave-core/pull/21796/commits/bce6b6fe47a88b5ec741e308f1c79f93d446e440)